FiveCalls/FiveCalls/IssueActivityItemSource.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  IssueActivityItemSource.swift
+//  FiveCalls
+//
+//  Created by Alex on 2/4/17.
+//  Copyright © 2017 5calls. All rights reserved.
+//
+
+import UIKit
+
+class IssueActivityItemSource: NSObject, UIActivityItemSource {
+    let issue: Issue
+    
+    init(issue: Issue) {
+        self.issue = issue
+    }
+    
+    func activityViewControllerPlaceholderItem(_ activityViewController: UIActivityViewController) -> Any {
+        return issue.name
+    }
+    
+    func activityViewController(_ activityViewController: UIActivityViewController, itemForActivityType activityType: UIActivityType) -> Any? {
+        return "I just called my rep to \(issue.name) — you should too: http://5calls.org/#issue/\(issue.id)?utm_campaign=twshare"
```

Comment:
@nickoneill just a heads up regarding the share message text.  Do you think this grammar assumption is okay to make?